json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
(I think we should as a matter of correctness, even though I can't think of a clear example of us validating briefs again after they're published.)